concord/computation.py

```diff
@@ -37,10 +35,13 @@
 import logging
 import logging.handlers
 
-logging.basicConfig()
-concord_logging_handle = logging.handlers.RotatingFileHandler("concord_py.log")
-concord_logger = logging.getLogger(__name__)
-concord_logger.setLevel(logging.DEBUG)
+logging.basicConfig(format='concord: %(name)s (%(levelname)-4s): %(message)s')
+concord_logging_handle = logging.handlers.RotatingFileHandler("concord_py.log",
+                                                              # 512MB
+                                                              maxBytes=512000000,
+                                                              backupCount=10)
+concord_logger = logging.getLogger('concord.computation')
```

Comment:
maybe we want to prefix with the project name. For example cmd.deploy or hangar.entrypoint